config/phobos.yml.example

```diff
@@ -1,6 +1,8 @@
 logger:
   # Optional log file, set to false or remove to disable it
   file: log/phobos.log
+  # Optional log format for stdout, remove to use human readable format
+  stdout_format: json
```

Comment:
Is there an explicit value for human readable format? That is, if I don't want to remove the configuration but rather explicitly set it to the default.

Reply:
Not really. Omitting the value (i e just updating phobos and not adding any line) will keep the old behaviour. But I agree it is weird to have what looks like an Enum but really only allows one value (`json`).

Perhaps a better nomenclature would be 
```yml
logger:
  # Optional log file, set to false or remove to disable it
  file: log/phobos.log
  # Optional output format for stdout, default is false (human readable). Set to true to enable json output.
  stdout_json: false
```